test/unit_test/static/quantity_point_test.cpp

```diff
@@ -37,7 +42,7 @@ namespace {
 
 using namespace units;
 using namespace isq::si;
-using namespace references;
+using namespace isq::si::references;
```

Comment:
Why did this work before? Oh, the `units::reference` in `generic/angle.h` wasn't visible before.